src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("6.0.3.0")]
-[assembly: AssemblyFileVersion("6.0.3.0")]
+[assembly: AssemblyVersion("6.1.0.0")]
```

Comment:
There is a breaking change, if the breaking change remains, the major version should be updated.  We should really avoid the breaking change in this case.  Aslo, please change this to the supported pattern, with Assembly version = 6.0.0.0,